Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,81 @@
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The Node.js Admin Repository (nodejs/admin) will serve the central location for
+managing Node.js GitHub Organization administrative activities. Only Node.js
+GitHub Organization owners will have write permissions to the nodejs/admin
+repository.
+
+## Organization Roles
+
+### Owners
+
+The Node.js Foundation Executive Director, Node.js TSC Director, TSC Chair,
+Community Committee Chair, and Node.js Foundation Education Manager shall be
+the only individuals granted Owner permissions within the Node.js GitHub
+Organization. Should, at any point in the future, the Node.js Foundation
+Board establish a Community Committee Director position equivalent to the
+TSC Director, that individual would also be automatically granted Owner
+permissions within the organization.
+
+Upon the completion of the terms of service for each of these individuals,
+their Owner permissions within the GitHub organization shall be removed.
+
+### Members
+
+GitHub users are added as members to the Node.js GitHub Organization when they
+are added to any Working Group. Organization Owners should add new members
+to the organization when requested by a Working Group.
+
+## Repositories
+
+Any organization member may request the creation of a new repository within the
+Node.js Foundation GitHub Organization. Provided there are no objections from
+any voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a vote in
+each of the Technical Steering and Community Committees.  A simple majority of
+each group *rejecting* the creation of the repository is required to block
+creating the repository. Such requests must be posted as issues in the
+nodejs/admin repository.
+
+Any repository created under the Node.js GitHub Organization is considered to be
+a project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation.
+
+No repository may be deleted, transferred in to, or transferred out of, the
+Node.js Foundation GitHub Organization without a simple majority of both
+the TSC and CommComm in *favor* of the action. In certain cases, Node.js
+Foundation Board of Directors approval may also be required.
+
+All repositories must have a README that clearly identifies the purpose of the
+repository, governance documentation that details how the repository is managed,
+and an indication of whether the repository falls under the operational
+direction of either the TSC or CommComm. 
+
+Repositories that are under the operational direction of the TSC are subject to
+TSC oversight.
+
+Repositories that are under the operational direction of the Community Committee
+are subject to CommComm oversight.
+
+## Removing or Banning Individuals
```

Comment:
I feel like removing and banning are overlapping with @nodejs/moderation team's responsibilities.

Reply:
No, only org owners can ban. If the moderation team determines that a ban is necessary, they would ask one of the owners to actually do the ban.

Reply:
Specifically, mod team does not have owner status in the org